packages/react-conformance-griffel/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@fluentui/react-conformance-griffel",
+  "version": "9.0.0-beta.4",
```

Comment:
nit: we are not releasing in lockstep, right?

Reply:
this is package version, has nothing to do with locksteping , or what do you mean?

Reply:
Good call, copy-paste issue 🤓 Downgraded to `9.0.0-beta.0`